recipe/meta.yaml

```diff
@@ -25,25 +24,27 @@ requirements:
     - setuptools
     - cython
     - libev  # [not win]
-    - msinttypes  # [win and py2k]
 
   run:
     - python
     - numpy
     - six >=1.9
-    - futures <=2.2.0  # [py2k]
-    - libev  # [not win]
```

Comment:
libev is not needed?

Reply:
@wolfv, thanks! I looked into this I'll add it back. It's optional but presumably desirable.